backend-api/src/functions/asyncAbortSession/validateRequestBody/validateRequestBody.ts

```diff
@@ -0,0 +1,36 @@
+import { validateSessionId } from "../../common/request/validateSessionId/validateSessionId";
+import { errorResult, Result, successResult } from "../../utils/result";
+
+export function validateRequestBody(
+  body: string | null,
+): Result<IAsyncAbortSessionValidParsedRequestBody> {
+  if (body == null) {
+    return errorResult({
+      errorMessage: `Request body is either null or undefined.`,
+    });
+  }
+
+  let parsedBody;
+  try {
+    parsedBody = JSON.parse(body);
+  } catch (error: unknown) {
+    return errorResult({
+      errorMessage: `Request body could not be parsed as JSON. ${error}`,
+    });
+  }
+
+  const { sessionId } = parsedBody;
+
+  const validateSessionIdResult = validateSessionId(sessionId);
+  if (validateSessionIdResult.isError) {
+    return validateSessionIdResult;
+  }
+
+  return successResult({
+    sessionId,
+  });
```

Comment:
suggestion (non-blocking): we are only sending back one thing here, so we could just do 

```suggestion
  return successResult(sessionId);
```

Reply:
Your interfaced and happy path test would need to change if you updated this

Reply:
Done